src/Compilers/CSharp/Test/WinRT/Metadata/WinMdDumpTest.cs

```diff
@@ -10,6 +10,7 @@
 using Microsoft.CodeAnalysis.CSharp.Symbols.Metadata.PE;
 using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Test.Utilities;
+using Roslyn.Test.Utilities;
```

Comment:
Nit: Can this be reverted?

Reply:
This is actually necessary given MetadataSignature moved from Utilities.Desktop to Utilities (it's no longer desktop specific). I will remove the .Desktop reference though as it's not needed. 

---
In reply to: [171425926](https://github.com/dotnet/roslyn/pull/25050#discussion_r171425926) [](ancestors = 171425926)